lms/djangoapps/instructor/views/data_access_constants.py

```diff
@@ -48,8 +49,8 @@ class Inclusion:
                          'N': Inclusion.NOT,
                          'F': Inclusion.FILTER}
 
-INCLUDE_SECTION_PATTERN = re.compile('chapter|sequential')
-INCLUDE_PROBLEM_PATTERN = re.compile('problem')
+INCLUDE_SECTION_PATTERN = re.compile('|'.join(settings.INSTRUCTOR_QUERY_SECTION_TYPES))
```

Comment:
Any xblock that Monica didn't think of at the time or any new one. There's a precedent for putting these types of lists in common (STUDENT_RESPONSES_REPORT_SUPPORTED_TYPE, INLINE_ANALYTICS_SUPPORTED_TYPES). Why would we not want to put this in common @stvstnfrd ?